conda/models/environment.py

```diff
@@ -162,7 +162,7 @@ def from_context(cls) -> EnvironmentConfig:
         field_names = {field.name for field in fields(cls)}
 
         environment_settings = {
-            key: value
+            key: list(value) if isinstance(value, tuple) else value
```

Comment:
hmmm, I'm not completely sure that this is a good idea. Is there a case when we want an `EnvironmentConfig` field to be a `tuple`? Probably?

Reply:
that approach sounds good to me!